src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/ConfigurationBinderBenchmarks.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class ConfigurationBinderBenchmarks
+    {
+        [Params(3, 6, 9)]
+        public int MyObjectCount { get; set; }
+
+        [Params(2, 4, 6)]
```

Comment:
Is it common to have so many duplicates? Perhaps we should benchmark lack of duplicates too?

https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#code-paths
```suggestion
        [Params(0, 2)]
```

Reply:
Removed the duplication. In the valid scenarios I have been working on, the configuration should always have unique keys, even across files.